reading-list.njk

```diff
@@ -1,5 +1,7 @@
 ---
-layout: episode-page.njk
+layout: layouts/main.njk
+title: Rehydrate - Reading List
+pageTitle: Reading List
```

Comment:
Do we want to swap `pageTitle` and `title` here?

Reply:
having the title having "rehydrate" in it seemed better for looking at history, etc.. having it display on the page seemed redundant when there's already the logo

Reply:
But which one is the one that goes in `<title>`? That should be `pageTitle`, right?

Reply:
I had it as `title`, but maybe it's better to rename to `title` and `pageHeader` to be more clear